packages/server/admin/controllers/tools.controller.js

```diff
@@ -0,0 +1,41 @@
+const knex = require('../../config/db');
+const HttpError = require('../../api/lib/utils/http-error');
+
+const createTool = async (body) => {
+  if (Object.keys(body).length === 0 || body === {}) {
+    throw new HttpError('Message: Bad Input', 422);
+  }
+  await knex('tools').insert({
```

Comment:
do we need to add validation for the calling user that they are admins? otherwise, I can just manually call this endpoint and add tools even though I am not an admin